DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateGroups/GroupCourses.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+@using Microsoft.Extensions.Configuration
+@model GroupCoursesViewModel
+
+@{
+  ViewData["Title"] = "Group courses";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/delegates/groupDelegates.css")" asp-append-version="true">
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+@section NavBreadcrumbs {
+  <partial name="_DelegateGroupBreadcrumbs" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter">
+    <nav class="side-nav-menu" aria-label="Side navigation bar">
+      <partial name="_DelegateGroupsSideNavMenu" model="Model.NavViewModel" />
+    </nav>
+  </div>
+
+  <div class="nhsuk-grid-column-three-quarters">
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-two-thirds">
+        <h1 id="page-heading" class="nhsuk-heading-xl">Group courses</h1>
+      </div>
+      <div class="nhsuk-grid-column-one-third delegate-heading-button-group">
+        <a class="nhsuk-button delegate-heading-button" role="button">
+          Add
+        </a>
+      </div>
+    </div>
+
+    <form method="get">
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-full">
+          @if (!Model.GroupCourses.Any())
+          {
+            <p class="nhsuk-u-margin-top-4" role="alert">
+              <b>No courses found.</b>
+            </p>
+          }
+          else
+          {
+            <div>
+              @foreach (var groupCourse in Model.GroupCourses)
+              {
+                <partial name="_GroupCourseCard" model="groupCourse" />
+              }
+            </div>
+          }
+          @if (Model.TotalPages > 1)
+          {
+            <partial name="SearchablePage/_Pagination" model="Model" />

```

Comment:
Given that this is not searchable page, should the pagination partial folder have a different name/should it live in a different folder (as it's more generic)?